packages/vscode-extension/src/commands/UploadFile.ts

```diff
@@ -67,9 +67,21 @@ export default class UploadFile extends BaseAction {
 				message: this.translationService.getMessage(ERRORS.NO_ACTIVE_WORKSPACE),
 			};
 		} else {
+			const projectInfoService = new ProjectInfoServive(this.executionPath);
+			if (projectInfoService.isAccountCustomizationProject() || projectInfoService.isSuiteAppProject) {
```

Comment:
I believe projectInfoService should be initialized, as David pointed out some lines above. That is, `projectInfoService.isSuiteAppProject()`.

Reply:
Updated.

Reply:
I still see `projectInfoService.isSuiteAppProject` and not `projectInfoService.isSuiteAppProject()`. Is that on purpose?

Reply:
Sorry about that. Updated again.